test/e2e/setup.go

```diff
@@ -377,9 +391,35 @@ var _ = BeforeSuite(func() {
 
 	if err := setup(context.Background()); err != nil {
 		panic(err)
+	} else {
```

Comment:
the else isn't necessary, since we panic in the block right before

Reply:
removed else statement.